spec/Phpro/SoapClient/BridgeBundle/Type/SoapCallSpec.php

```diff
@@ -47,7 +48,26 @@ function it_should_stop_request(
         RequestEvent $requestEvent,
         ResultInterface $result,
         Client $client
-    ) {
+    )
+    {
+
+        $debugData = [
+            'request' => [
+                'headers' => 'SoapRequestHeader',
+                'body' => '<?xml version="1.0" encoding="UTF-8"?><body>SoapRequestBody</body>'
+            ],
+            'response' => [
+                'headers' => 'SoapResponseHeader',
+                'body' => '<?xml version="1.0" encoding="UTF-8"?><body>SoapResponseBody</body>'
+            ],
+        ];
+
+        $client->debugLastSoapRequest()->willReturn($debugData);
+
+        $requestEvent->getRequest()->shouldBeCalled();
```

Comment:
These should also return something instead of validating wether the method was called.